### DISCUSSION TITLE
(Automatic) Onscreen Keyboards for containers

### DISCUSSION THREAD

**Discussion by PfiatDe** [accetto/ubuntu-vnc-xfce-g3#54] [Q&A]
Using a Ubuntu LTS 22.04 container with firefox and noVNC. This is working great so far.
However, if the connecting client is a mobile device, it gets almost impossible to use, as there is no onscreen keyboard to type.
I want to avoid to blend in the NoVNC control panel with the keyboard button. It would be better to rely on either 
- the onscreenkeyboard of the device, but therefore the input event must somehow be forwarded and I have no idea how
- the onscreenkeyboard of the OS (Ubuntu). However I did not find a way to automatically open an onscreen keyboard on ubuntu like on a mobile device.

Anybody got an idea, how to solve this?

**Comment by accetto** [comment]
Hello,

thank you for your contribution. Unfortunately I have no idea if such a thing is possible. Maybe somebody would know.

Regards,
accetto
